types/src/on_chain_config/aptos_features.rs

```diff
@@ -274,6 +279,29 @@ impl Features {
     pub fn is_refundable_bytes_enabled(&self) -> bool {
         self.is_enabled(FeatureFlag::REFUNDABLE_BYTES)
     }
+
+    pub fn get_max_identifier_size(&self) -> u64 {
+        if self.is_enabled(FeatureFlag::LIMIT_MAX_IDENTIFIER_LENGTH) {
+            IDENTIFIER_SIZE_MAX
+        } else {
+            LEGACY_IDENTIFIER_SIZE_MAX
+        }
+    }
+
+    pub fn get_max_binary_format_version(&self, gas_feature_version_opt: Option<u64>) -> u32 {
```

Comment:
maybe_gas_feature_version (for optional)? seems like a more common convention that's been used in the codebase.

Reply:
Actually, it is always Some(), changed to use u64 directly.